images/amd64/appdev/qemu/crinit/config_root.sh

```diff
@@ -6,6 +6,10 @@ chmod +x ./sbin/*
 # Create a fake machine-id
 echo "04711" > ./etc/machine-id
 
+# allow name resolution (needed for apt update)
+touch /etc/resolv.conf
+echo "nameserver 1.1.1.1" > /etc/resolv.conf
```

Comment:
/etc/resolv.conf should be a symlink to the resolv.conf generated by netifd.

Reply:
is there an example or can you please give a suggestion like?
```suggestion
ln -s <netifd>resolv.conf /etc/resolv.conf
```

Reply:
The resov.conf created by netfifd is /run/resolv.conf.netifd as defined in https://github.com/Elektrobit/ebcl_template/blob/main_1.x/images/amd64/appdev/qemu/crinit/config/etc/crinit/crinit.d/network.crinit

It is also possible to change the crinit unit, to directly write the resolv.conf to /etc. In the example where the crinit unit was copied from used a path in /var, because /etc was on a readonly filesystem.